mapper/src/main/scala/org/scalarelational/mapper/Entity.scala

```diff
@@ -7,17 +10,19 @@ import org.scalarelational.instruction.{Update, InsertSingle}
  * @author Matt Hicks <matt@outr.com>
  */
 trait Entity {
-  def toColumnValues: List[ColumnValue[Any]] =
-    throw new RuntimeException(s"@mapped annotation missing for table ${this.getClass}")
+  def mapTo[T <: Entity](table: Table): List[ColumnValue[Any]] =
+    macro mapped.mapTo[T]
+
+  def columns: List[ColumnValue[Any]]
```

Comment:
I agree. The reason is that it behaves like a DSL: ``def columns = mapTo[Supplier](Datastore.suppliers)``. How would you name it?